pkg/labels/labels_test.go

```diff
@@ -368,21 +368,7 @@ func TestLabels_Equal(t *testing.T) {
 	}
 }
 
-func TestLabels_FromStrings(t *testing.T) {
```

Comment:
This test is duplicated below, so I've removed it. I only kept the test for panic.

Reply:
The duplicate test got removed in https://github.com/prometheus/prometheus/pull/7461, so we can revert this change after rebasing

Reply:
Sure, done.